semgrep-core/tests/yaml/yaml_rules/kubernetes_semgrep.yaml

```diff
@@ -0,0 +1,19 @@
+rules:
+- id: kubernetes.best-practice.no-fractional-cpu-limits
+  patterns:
+  - pattern-inside: |
+      limits: ...
```

Comment:
This another example that does not work @emjin . It works with spacegrep (see the kubernetes_spacegrep.yaml file in the same directory) but not semgrep.
Looks more complicated to fix though, nothing obviously wrong, the AST seemed ok.